src/codegate/db/models.py

```diff
@@ -37,6 +38,12 @@ class Setting(pydantic.BaseModel):
     other_settings: Optional[Any]
 
 
+class Workspace(pydantic.BaseModel):
+    id: Any
+    name: str
```

Comment:
Just an idea, I wonder if we may want to use the org: name: format. For example, I could then sort views of repositories via the stacklok org (and see codegate,minder etc)